sidekiq/metadata.csv

```diff
@@ -7,13 +7,16 @@ sidekiq.jobs.perform.95percentile,gauge,,,,95th percentile of amount of time spe
 sidekiq.jobs.count,count,,,,Count of Sidekiq jobs,0,sidekiq,jobs count
 sidekiq.jobs.success,count,,,,Count of successful Sidekiq jobs,0,sidekiq,jobs success
 sidekiq.jobs.failure,count,,,,Count of failed Sidekiq jobs,-1,sidekiq,jobs failure
```

Comment:
IIUC those are totals:

```
sidekiq.jobs.count,count,,,,Count of Sidekiq jobs,0,sidekiq,jobs count
sidekiq.jobs.success,count,,,,Count of successful Sidekiq jobs,0,sidekiq,jobs success
sidekiq.jobs.failure,count,,,,Count of failed Sidekiq jobs,-1,sidekiq,jobs failure
```


And we should add per worker metrics.

Reply:
Updated!